css/main.css

```diff
@@ -92,21 +160,17 @@ h2 {
 	padding-top: 35px;
 }
 
-.sights-heading {
-	clear: both;
-	padding-top: 20px;
-
-}
-
 input[type=radio]+label,
```

Comment:
please don't use selectors like a tag name, use classes instead. It will help you to avoid conflicts with the other pages.